battenberg_fresh.yaml

```diff
@@ -0,0 +1,219 @@
+name: battenberg-fresh
```

Comment:
I don't see a reference to this environment

Reply:
Where is this file? I can't tell what the path is and I don't see it in my local repo

Reply:
It does not show a path, only a file name. This file shows as "Outdated" now. Maybe it is something with git, and this file has been deleted/mowed since then

Reply:
I found it. I had to "pull" to the repo. It's now deleted